packages/edit-site/src/components/block-editor/editor-canvas.js

```diff
@@ -87,7 +87,9 @@ function EditorCanvas( { enableResizing, settings, children, ...props } ) {
 					// which isn't a requirement in auto resize mode.
 					enableResizing ? 'min-height:0!important;' : ''
 				}}body{position:relative; ${
-					canvasMode === 'view' ? 'cursor: pointer;' : ''
+					canvasMode === 'view'
+						? 'cursor: pointer; height: 100vh'
```

Comment:
I'm not sure 100vh is the right way to go about this but `min-height:100%` isn't working and I don't understand why. If you manually add something like `border: 10px solid yellow;` to the body you will see scrollbars. I don't think this should block your PR but I'm going to investigate why min-height isn't working with how the HTML element is styled.

Reply:
Ok I see why, html needs to have height:100% instead of height: auto; for the min-height on body to work. But that produces scrollbars anyway on the edge case that I mentioned, so let's forget I said anything and go with your solution then. I doubt we will see many cases where this happens.